masterlist.yaml

```
@@ -22981,17 +22981,24 @@ plugins:
         nav: 4
 
   - name: 'WindhelmEntranceOverhaul.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19400/' ]
+    url: 
```

There's an extra whitespace here:

```suggestion
    url:
```